.changeset/quick-forks-nail.md

```
@@ -0,0 +1,5 @@
+---
+"@jsandy/rpc": minor
+---
+
+Added support for getting the schema of the rpc routers
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)